src/components/TheHeader.vue

```diff
@@ -29,12 +32,10 @@ export default {
     },
   },
 
-  setup() {
-    const { list, total } = useIncidents()
-
-    const allCities = computed(() => [...new Set([...list.value.map((i) => i.city)])].sort())
-
-    return { total, allCities }
+  computed: {
+    showFilter() {
+      return this.$route.name === 'City' || this.$route.name === 'Root'
```

Comment:
Interesting, it's surprising to me that `this.$route` is available here given this component is no longer a child of `<router-view>`. It means `<TheHeader />` is now coupled to the router as well as `<incident-filter />`.

Anyway, we chatted on Slack; the coupling doesn't matter for our tiny app anyway 👍

Reply:
this is fixed